### DISCUSSION TITLE
Full width Grid columns

### DISCUSSION THREAD

**Discussion by rovashan** [mantinedev/mantine#4636] [Q&A] [answered]
Hi, I'm using the Grid component with responsive columns. The responsiveness is working fine but I would like to have each column take the full width of the parent. I believe that using a minWidth property will break the responsiveness. Any suggestions or is there another component that I can use to achieve this?

```
  <Grid gutter="lg" justify="center">
      <Grid.Col md={6} lg={6}>
          <Paper
              radius="md"
              withBorder
              p="lg"
          >
              Grid                        
          </Paper>
      </Grid.Col>

      <Grid.Col md={6} lg={6}>
          <Paper
              radius="md"
              withBorder
              p="lg"
          >
              Grid                    
          </Paper>
      </Grid.Col>
  </Grid>
```

<img width="806" alt="image" src="https://github.com/mantinedev/mantine/assets/4746551/d44cc0d5-482b-4cc5-8316-41062c636665">

**Comment by rtivital** [comment]
To make columns full width, set column size at breakpoint to `12`, for example `md={12}`

**Reply by rovashan** [reply]
I would like to keep two columns using md={6} but have each column fill the available width, irrespective of the content size.

Example, I achieved the result below by setting the width property but then it's no longer responsive.

<img width="1084" alt="image" src="https://github.com/mantinedev/mantine/assets/4746551/dbd2aa27-c510-4aee-8e45-987166009633">

**Reply by rtivital** [reply, marked as answer]
Columns take all available space – https://codesandbox.io/s/polished-glitter-82ry3k?file=/src/App.tsx
If they do not, it means that you have styles in your application that prevent that. For example, display:flex on parent element.

**Reply by rovashan** [reply]
That was it - I had display:flex on a parent element.

Thank you for your help and for a beautiful library!
